openapi.yaml

```
@@ -710,6 +710,79 @@ paths:
                   $ref: "#/components/schemas/Budget"
         "500":
           $ref: "#/components/responses/InternalServerError"
+  /api/rooms:
+    get:
+      summary: 部屋の一覧を取得
+      tags:
+        - Rooms
+      operationId: getRooms
+      responses:
+        "200":
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/Room"
+        "500":
+          $ref: "#/components/responses/InternalServerError"
+    post:
+      summary: 部屋を作成
+      tags:
+        - Rooms
+      operationId: postRoom
+      parameters:
+        - $ref: "#/components/parameters/X-Forwarded-User"
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/PostRoomRequest"
+      responses:
+        "201":
+          description: Created
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Room"
+        "400":
+          $ref: "#/components/responses/BadRequest"
+        "403":
+          $ref: "#/components/responses/Forbidden"
+        "409":
+          $ref: "#/components/responses/Conflict"
+        "500":
+          $ref: "#/components/responses/InternalServerError"
+    put:
```

これは僕がミスってたんですが、PUTの場合はどの部屋に対する操作かを指定できる必要があります。`parameters`にも`roomId`みたいな項目を追加する必要がありそうです
```suggestion
  /api/rooms/{room_id}
    put:
```